<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generator</class>
 <widget class="QMainWindow" name="generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Button CSS Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/img/assests/app.svg</normaloff>:/img/assests/app.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>638</width>
         <height>532</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Font/Text</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="ff_label">
             <property name="text">
              <string>Font-family :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFontComboBox" name="ff_input"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="fc_label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color &lt;a href=&quot;...&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8aadf4;&quot;&gt;Pick&lt;/span&gt;&lt;/a&gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="fc_input">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="fs_label">
             <property name="text">
              <string>Font-size :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="fs_input">
             <property name="currentIndex">
              <number>5</number>
             </property>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>22</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>24</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>26</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>28</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>36</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>48</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>72</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Box</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="pt_label">
             <property name="text">
              <string>Padding-top :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="pt_input"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="pr_label">
             <property name="text">
              <string>Padding-right :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="pr_input"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="pb_label">
             <property name="text">
              <string>Padding-bottom :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="pb_input"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="pl_label">
             <property name="text">
              <string>Padding-left :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="pl_input"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Border</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="2" column="0">
            <widget class="QLabel" name="br_label">
             <property name="text">
              <string>Border-radius :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="br_input"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="bw_label">
             <property name="text">
              <string>Border-width :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="bw_input"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="bc_label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color &lt;a href=&quot;...&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8aadf4;&quot;&gt;Pick&lt;/span&gt;&lt;/a&gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="bc_input"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="bs_label">
             <property name="text">
              <string>Border-style :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="bs_input">
             <item>
              <property name="text">
               <string>Solid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dotted</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dashed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Groove</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Double</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ridge</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Inset</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Outset</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Background</string>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="bkc_label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color &lt;a href=&quot;...&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8aadf4;&quot;&gt;Pick&lt;/span&gt;&lt;/a&gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="bkc_input"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Hover</string>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="bkch_label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color &lt;a href=&quot;...&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8aadf4;&quot;&gt;Pick&lt;/span&gt;&lt;/a&gt; :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="bkch_input">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maxLength">
              <number>32767</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="gen_css_btn">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Generate CSS</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionJoin_us_on_twitter"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionJoin_us_on_twitter">
   <property name="text">
    <string>Follow me on twitter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
